pkg/ocm/ocm_test.go

```diff
@@ -0,0 +1,44 @@
+package ocm
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	sdkcfg "github.com/openshift-online/ocm-cli/pkg/config"
+)
+
+var _ = Describe("OCM", func() {
+	Describe("When trying to load a configuration", func() {
+		var (
+			config         *sdkcfg.Config
+			configLocation string
+		)
+		JustBeforeEach(func() {
```

Comment:
I understand how it works, but maybe this should be explained for future developers

Reply:
Done